src/Components/Blazor/Templates/src/content/BlazorHosted-CSharp/Client/Pages/Counter.razor

```diff
@@ -4,7 +4,7 @@
 
 <p>Current count: @currentCount</p>
 
-<button class="btn btn-primary" @onclick="IncrementCount">Click me</button>
+<button class="btn btn-primary" @onclick="@IncrementCount">Click me</button>
```

Comment:
In Preview 7 the second `@` isn't needed.

```suggestion
<button class="btn btn-primary" @onclick="IncrementCount">Click me</button>
```

Reply:
ok